### PR TITLE
provider: change bradesco endpoint config and add log.

### PR DESCRIPTION
**Pull request by Thor99**
## Description

- Change provider production config to change endpoint based on STAGE variable.

- Add log.

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](../CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation
1. I feel confortable with the code I wrote, and I'm not ashamed to show it to my friends

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: src/providers/bradesco/index.ts [hunk 1]
```diff
@@ -56,8 +56,12 @@ export const buildPayload = boleto =>
     .then(spec => applySpec(spec)(boleto))
 
 export const translateResponseCode = (response) => {
+  const logger = makeLogger({ operation: 'translateResponseCode' })
+
   const responseCode = response.data.status.codigo.toString()
 
+  logger.info({ status: 'succeeded', metadata: { providerResponse: response.toString() } })
```
Comment (mccraveiro) [inline src/providers/bradesco/index.ts]: 😢 nem deram bola para meu review

Comment (felquis) [inline src/providers/bradesco/index.ts]: Shit happends

Comment (grvcoelho) [inline src/providers/bradesco/index.ts]: @mccraveiro, @felquis: levamos sim seu review em consideração, mas por não quebrar nenhuma regra do linter que usamos neste projeto, decidimos que não precisaríamos da alteração :)

Comment (mccraveiro) [inline src/providers/bradesco/index.ts]: https://github.com/pagarme/javascript-style-guide/issues/10

**Pull request merged**